resources/public/api/conf/2.0/application.yaml

```diff
@@ -1104,16 +947,10 @@ paths:
           content:
             application/json:
               schema:
-                allOf:
-                  - $ref: '#/components/schemas/NotFound'
-                  - example:
-                      code: NOT_FOUND
-                      message: Supplied arrival not found or does not exist or has been archived or is not available to the EORI number.
-                               Not found if the page requested is greater than the total number of pages.
+                $ref: '#/components/schemas/NotFound'
               example:
                 code: NOT_FOUND
-                message: Supplied arrival not found or does not exist or has been archived or is not available to the EORI number.
-                         Not found if the page requested is greater than the total number of pages.

```

Comment:
There is no pagination on this endpoint (GET arrivals/{arrivalID}) so I removed this part from the examples.